README.md

```diff
@@ -1,8 +1,15 @@
-# README for STM32 custom targets
-
 Table of Contents
 =================
 
+* [Usage](#usage)
+* [STM32F1](#stm32f1)
+   * [BLUEPILL](#bluepill)
+* [STM32WL](#stm32wl)
+* [Customer boards](#customer-boards)
```

Comment:
```suggestion
* [Custom boards](#custom-boards)
```

Reply:
It will overwritten with https://github.com/ARMmbed/stm32customtargets/pull/1/files